packages/animations/test/open_container_test.dart

```diff
@@ -19,7 +19,7 @@ void main() {
 
       await tester.pumpWidget(_boilerplate(
         child: Center(
-          child: OpenContainer(
+          child: OpenContainer<dynamic>(
```

Comment:
for all containers that don't return anything, the correct type would be `void`, not dynamic.

Reply:
Since `@optionalTypeArgs` is used, the default type argument is `Object`. Is there any way to set the default type argument to `void` instead of `Object`?